tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
+"""Check modules from Ouroboros that compile with voc."""
+
+# Standard library imports
+from datetime import datetime
+from subprocess import Popen, PIPE
+import multiprocessing
+import os
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+MODULES_WHITELIST = [
```

Comment:
@freakboy3742 are the modules inside python subdirectory following the same structure, so that I can add the override mode based on that?